test/e2e/testdata/config/localkeys/e2e.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can we create this cert dynamically? This way we can run test with different variations of certificates.

Reply:
It is not necessary to create it dynamically. multiple pre-baked certificates file can be keeped in the repo. Also we need to validate some cases with pre-signed artifact, so we need the same certificate to verify it.